cmd/juju/controller/listmodels_test.go

```diff
@@ -102,12 +106,21 @@ func (f *fakeModelMgrAPIClient) ListModelsWithInfo(user names.UserTag) ([]params
 			AgentVersion:       info.Result.AgentVersion,
 		}
 		if len(info.Result.Users) > 0 {
-			results[i].Result.User = info.Result.Users[0]
+			results[i].Result.UserAccess = info.Result.Users[0].Access
+			results[i].Result.UserLastConnection = info.Result.Users[0].LastConnection
```

Comment:
This feels like we should be changing something in one of the layers to remove this sort of friction.